packages/openapi-fetch/test/index.test.ts

```diff
@@ -859,6 +855,138 @@ describe("client", () => {
       });
     });
 
+    describe("content-type", () => {
+      const BODY_ACCEPTING_METHODS = [["PUT"], ["POST"], ["DELETE"], ["OPTIONS"], ["PATCH"]] as const;
```

Comment:
nit: Is there a reason to have two dims arrays here?

Reply:
This is what the [each method](https://vitest.dev/api/#test-each) officially expects. It's a list of param sets where each set can consist of multiple params. The API supports a flat array, where `[x]` is treated as `[[x]]`, but I don't really fancy these magical interfaces and prefer to be explicit. Imagine what happens when `x` is actually an array?